CONTRIBUTING.md

```diff
@@ -1,4 +1,49 @@
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
+
 ## Contributing to this Project
 
-OpenSearch is a community project that is built and maintained by people just like **you**.
+    OpenSearch is a community project that is built and maintained by people just like **you**.
 [This document](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md) explains how you can contribute to this and related projects.
+The brief summary below outlines contribution guidelines specific to the OpenSearch-Security Repo.
+
+## Triaging
+
+OpenSearch-Security seeks to promote an inclusive and engaged community of contributors. In order to facilitate this, weekly triage meetings are open-to-all and attendance is encouraged for anyone who hopes to contribute, discuss an issue, or learn more about the project.
+
+FAQ about OpenSearch-Security Triage Meetings:
```

Comment:
Raise to H2 level to nest all the H3's below under

Reply:
I think these questions are visible and useful on their own, maybe consider dropping the line?

Reply:
Dropping what line?

Reply:
> FAQ about OpenSearch-Security Triage Meetings: